lib/components/widgets/loginScreen/login_screen.dart

```diff
@@ -0,0 +1,134 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/gestures.dart';
+import 'package:classroom_app/components/styles/color/colors.dart';
+import 'package:classroom_app/components/widgets/CustomButton/custom_button.dart';
+import 'package:classroom_app/components/widgets/SignUpScreen/sign_up_screen.dart';
+
+class LoginScreen extends StatelessWidget {
+  const LoginScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.of(context).size;
+    final screenWidth = size.width;
+    final screenHeight = size.height;
+    return Scaffold(
+      backgroundColor: AppColors.white,
+      body: LayoutBuilder(
+        builder: (context, constraints) {
+          final isSmallScreen = constraints.maxHeight < 600;
+
+          return Padding(
+            padding: EdgeInsets.symmetric(horizontal: screenWidth * 0.05),
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                // Top spacing or logo section
+                if (!isSmallScreen)
+                  SizedBox(height: constraints.maxHeight * 0.1),
```

Comment:
Wrap the TextField widgets in a Form widget and manage their states using GlobalKey<FormState>. This will help in validating user input before login.
```
final _formKey = GlobalKey<FormState>();

@override
Widget build(BuildContext context) {
  return Form(
    key: _formKey,
    child: Column(
      children: [
        // Your other widgets...
      ],
    ),
  );
}
```